### ISSUE TITLE
[Bug]: Nextcloud Desktop Client is crashing and freezing in the cases of Windows 11 and virtual file support enabled feature

### ISSUE THREAD

**Issue by SignFinder** [nextcloud/desktop#6357]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

When Nextcloud Desktop Client virtual file support feature is enabling on Windows 11 OS clients  - nexcloud client starting to crash, freeze etc.
Crashes generate the events id 1000 like this:
Faulting application name: nextcloud.exe, version: 3.11.0.46123, time stamp: 0x65776699
Faulting module name: nextcloudsync.dll, version: 0.0.0.0, time stamp: 0x6577664b
Exception code: 0xc0000005
Fault offset: 0x0000000000019690
Faulting process id: 0x0x4438
Faulting application start time: 0x0x1DA49CE0A09C24F
Faulting application path: C:\Program Files\Nextcloud\nextcloud.exe
Faulting module path: C:\Program Files\Nextcloud\nextcloudsync.dll
Report Id: 658a8399-32e8-4ffe-a3bb-eb8fc30ac6ad
Faulting package full name: 
Faulting package-relative application ID: 

### Steps to reproduce

1. Take computer with Windows 11 (tested on 23H2)
2. Install Nextcloud Desktop Client 3.11
3. Login to selfhosted nextcloud and all works well.
4. Enable virtual file support feature and receive crashes and freezes of Nextcloud Desktop Client

### Expected behavior

Client should work and do not freeze and crash

### Which files are affected by this bug

nextcloudsync.dll ?

### Operating system

Windows

### Which version of the operating system you are running.

Windows 11 23H2

### Package

Appimage

### Nextcloud Server version

28.0.1

### Nextcloud Desktop Client version

3.11

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by joshtrichards** [comment]
> When Nextcloud Desktop Client virtual file support feature is enabling on Windows 11 OS clients - nexcloud client starting to crash, freeze etc.

I notice you said *clientS* here - are you saying you have seen/reproduced this on multiple Windows 11 devices?

**Comment by TigerwareDev** [comment]
I have the same issue with using windows 10 22H2. Also have the encryption module enabled.
Just started with NextCloud yesterday, and the windows client was never responsive, unless it did not have any sync folders configured.
I also tried it with one bigger folder and it only managed to uploaded about 145mb over night. So it did something, just not great.

Now I disabled virtual folders and the app became way more responsive and already uploaded 1,4GB in 10mb.
So that a great help, thanks!

Its still not perfect, like it crashed once a minute after turning all virtual folders off.
But its finally usable! Thanks again @SignFinder

**Comment by SignFinder** [comment]
> > When Nextcloud Desktop Client virtual file support feature is enabling on Windows 11 OS clients - nexcloud client starting to crash, freeze etc.
> 
> I notice you said _clientS_ here - are you saying you have seen/reproduced this on multiple Windows 11 devices?

Yes, we confirmed problem on several computers, unfortunately including Windows 10 22H2 x64 too. Nextcloud is freezing for 15-30 minutes after setup of the new nextcloud account !!! ONLY WITH virtual file support enabled feature!!! .

**Comment by coldacid** [comment]
Can also confirm for both Win10 and Win11. Like @SignFinder I have group folders installed and active on my Nextcloud instance although my instance is using the internal user backend and has server encryption disabled.

**Comment by GrgMdmn** [comment]
I can also confirm still having freezes when nextcloud app is enabled on my Windows 11 OS. No problem on Kubuntu (Linux) on my other computer.
